Modules/Microsoft365DSC/DSCResources/MSFT_SPOHubSite/MSFT_SPOHubSite.psm1

```diff
@@ -564,14 +564,19 @@ function Export-TargetResource
     $ConnectionMode = New-M365DSCConnection -Platform 'PnP' `
                 -InboundParameters $PSBoundParameters
 
-    $hubSites = Get-PnPHubSite
+    [array]$hubSites = Get-PnPHubSite
```

Comment:
what does this array do? :)

Reply:
it forces the result to be an array. if there was one object then without explicitly stating that this is an array the resulting type would be of that object and not a collection. This is a hail mary solution because there were some similar issues in the past when dealing with data returned from the cmdlets.